pallets/substrate-app/src/lib.rs

```diff
@@ -472,6 +485,40 @@ pub mod pallet {
             );
             Ok(())
         }
+
+        #[pallet::call_index(9)]
+        #[pallet::weight(<T as Config>::WeightInfo::register_erc20_asset())]
+        pub fn set_minimum_xcm_incoming_asset_count(
```

Comment:
And what about unit tests / migrations / benchmarks ?

Reply:
it will be done in a separate task